vue.config.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
 		},
 		optimization: {
 			splitChunks: undefined,
+			minimize: !TARGET_NODE, // needed for comparison of `.constructor.name`s
```

Comment:
Maybe [namedModules](https://medium.com/webpack/webpack-4-mode-and-optimization-5423a6bc597a#166e) or namedChunks would suffice?